src/CommonITILValidation.php

```diff
@@ -701,6 +701,23 @@ public static function getStatusColor($value)
         return $style;
     }
 
+    /**
+     * Get Ticket validation approver link
+     *
+     * @param string    $itemtype
+     * @param integer   $items_id
+     **/
+    public static function getApprover($itemtype, $items_id)
+    {
+        if (empty($items_id)) {
+            return null;
+        }
+
+        $approver = $itemtype::getById($items_id);
```

Comment:
`$approver` can be `null`, this case should be handled to prevent any fatal error.